stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -51,4 +95,35 @@ private static String getIdAttribute(View element) {
   public View getViewForHighlighting(Object element) {
     return (View)element;
   }
+
+  private class ElementContext {
+    private View mElement;
+
+    public void hook(View element) {
```

Comment:
If we're not actually doing anything in `hook()` or `unhook()`, and we're not keeping around any context or state for this `View`, then we don't need to have an `ElementContext` either. `onSetAttributesAsText()` then doesn't need to forward to `ElementContext.setAttributesAsText()` and can just inline that code. 

It's actually very important to try and avoid utilizing an `ElementContext` for the base `View` class because then we'll have an extra allocation for every single `View` in the system! :)